strings.txt

```diff
@@ -16217,48 +16217,644 @@
 
   [editor_storey_number]
     tags = ios, android
-    en = Storey number (maximum %d)
+    en = Number of floors (max %d)
     ru = Количество этажей (максимум %d)
+    fr = Nombre d'étages (max %d)
+    da = Antal etager (maks %d)
+    id = Jumlah lantai (maksimum %25)
+    ko = 층 수(최대 %d)
+    sv = Antal våningar (max %d)
+    tr = Kat sayısı (en fazla %d)
+    uk = Кількість поверхів (максимум %d)
+    vi = Số tầng (tối đa %d)
+    hu = Az emeletek száma (legfeljebb %d)
+    de = Anzahl der Etagen (%2d maximal)
+    fi = Kerrosten määrä (maks. %2d)
+    cs = Počet poschodí (max. %d)
+    it = Numero di piani (massimo %d)
+    nb = Antall etasjer (maks. %d)
+    zh-Hans = 楼号 (最大 %d)
+    th = จำนวนชั้น (มากสุด %d)
+    ja = フロアの数（最大％d）
+    ro = Număr de etaje (max %d)
+    ar = عدد الطوابق (بحد أقصى %d)
+    sk = Počet poschodí (max %d)
+    es = Número de plantas (máx. %d)
+    pl = Liczba pięter (maks. %d)
+    nl = Aantal verdiepingen (max. %d)
 
   [error_enter_correct_storey_number]
     tags = ios, android
-    en = Maximum building levels number is limited to %d.
-    ru = Редактируйте здания высотой максимум %d этажей.
-  
+    en = Edit the building with maximum 25 floors
```

Comment:
Игорь? 
Вообще такие вещи надо отдельно в табличке смотреть и заранее вычитывать, править в текстовом файле вот это всё жутко неудобно.

> On Apr 26, 2016, at 4:16 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61082508>:
> 
> >  
> >    [error_enter_correct_storey_number]
> >      tags = ios, android
> > -    en = Maximum building levels number is limited to %d.
> > -    ru = Редактируйте здания высотой максимум %d этажей.
> > -  
> > +    en = Edit the building with maximum 25 floors
> Даже лучше levels вместо floors, чтобы везде было консистентно.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295#r61082508>